src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -76,6 +78,8 @@ public static void init(final IForgeRegistry<Block> registry)
         blockRack = new BlockMinecoloniesRack().registerBlock(registry);
         blockTimberFrame = new BlockTimberFrame().registerBlock(registry);
         blockWayPoint = new BlockWaypoint().registerBlock(registry);
+        blockHutBarracks = (BlockHutBarracks) new BlockHutBarracks().registerBlock(registry);
+        blockHutBarracksTower = (BlockHutBarracksTower) new BlockHutBarracksTower().registerBlock(registry);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'blockHutBarracksTower' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)